html/browsers/windows/browsing-context-names/browsing-context-choose-parent-001.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>HTML Test: Choose browsing context - '_parent'</title>
+<link rel="author" title="Intel" href="http://www.intel.com/">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<div id="log"></div>
+<iframe id="embedded" src="resources/parent-iframe-1.html" name="parentWin" style="display:none"></iframe>
```

Comment:
Hah! That's the reason I had the hard-to-read assignment of `src` to that `iframe` in the script and not in the markup initially. OK, I'll rectify.